symmer/symplectic/base.py

```diff
@@ -775,6 +775,59 @@ def to_sparse_matrix(self) -> csr_matrix:
         )
         return sparse_matrix
 
+    def conjugate_op(self, R: 'PauliwordOp') -> 'PauliwordOp':
```

Comment:
new method! Sadly slower than just multiplying using symmer....

Maybe there is a better way of doing this?